pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -444,21 +444,41 @@ func (m *kubeGenericRuntimeManager) computePodContainerChanges(pod *v1.Pod, podS
 	}
 	changes.InitFailed = initFailed
 
+	podRef, err := ref.GetReference(api.Scheme, pod)
+	if err != nil {
+		glog.Errorf("Couldn't make a ref to pod %q: '%v'", format.Pod(pod), err)
+	}
+
 	// check the status of containers.
 	for index, container := range pod.Spec.Containers {
 		containerStatus := podStatus.FindContainerStatusByName(container.Name)
-		if containerStatus == nil || containerStatus.State != kubecontainer.ContainerStateRunning {
+		if containerStatus == nil {
+			changes.ContainersToStart[index] = fmt.Sprintf("Starting container %+v.", container)
+			continue
+		} else if containerStatus.State != kubecontainer.ContainerStateRunning {
+			// TODO: is there a better place to report failed containers?
```

Comment:
Ideally I would like to centralize these messages such that we could have ~=.rc file that could be internationalized. 

/cc @brendandburns